.github/workflows/install-python-gdal.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
This file primarily supports a future CI change, which is why I placed it in `.github`. It also is not necessarily appropriate to run in all environments, so I think it is worth isolating a bit.